CHANGELOG.rst

```diff
@@ -3,10 +3,14 @@ Changelog
 
 This document describes changes between each past release.
 
-2.7.0 (unreleased)
+2.6.2 (2015-09-09)
 ------------------
 
-- Nothing changed yet.
+**Bug fixes**
+
+- Expose CORS headers on subrequest error response and for non service errors (#435).
+- Make sure we pass a tuple to Postgresql list comparisons (#443).
+- Use the inheritance tree to select shared records in collection list (#444).
```

Comment:
inheritance tree is unknown in *cliquet*.  authorization ``get_bound_permissions`` callback ?

we may allow ourself to list this as an internal change ?